### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by adarachel**
## Implemented the following requirements:
- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder.
- The index.html file must be set as a template using the HTML Webpack Plugin.
- Create an index.js file and set an array of some simple to do tasks (array of objects).
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (thecodechaser) [inline package.json:9]: - [ ] I'm not able to run the application, There's no script to run the development server. Please make sure to add that and the app should start with no error.

File: modules/tasks.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        completed: false,
+        index: tasks.length,
+      };
+      tasks.push(task);
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+      addTask.value = '';
+    }
+  }
+};
+
```
Comment (thecodechaser) [inline modules/tasks.js:15]: - [ ] Please remove all functionalities of the application to add, remove and edit tasks. The requirements for this project are to only create structure and add hardcoded tasks and iterate over the tasks array and populate an HTML list item element for each task.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @adarachel👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (adarachel) [reply package.json]: todo

when i open the project with index.html in 'dist' folder i can see the app

Comment (thecodechaser) [reply package.json]: > todo
> 
> when i open the project with index.html in 'dist' folder i can see the app

@adarachel Hi, The app should run on the webpack server. we use dist folder for production

Comment (adarachel) [reply package.json]: todo2

so now my project is running from http://localhost:8081/ Please is that correct??

Comment (thecodechaser) [reply package.json]: > todo2
> 
> so now my project is running from http://localhost:8081/ Please is that correct??

@adarachel Yes

Comment (adarachel) [reply package.json]: thank you so much for your help!!!

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi @adarachel,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**